### PR TITLE
Fix `go vet`: Struct field tag issues

### LINKED ISSUE #5763: `go vet`: Struct field tag issues
## I'm submitting a ...

-  bug report

## Traffic Control components affected ...

-  Traffic Ops
-  Traffic Monitor
-  Traffic Router tester (Go)

## Current behavior:

`go vet -structtag ./...` finds 9 issues:

<table><thead><th>Message</th><th>Line of code</th></thead><tbody>

<tr><td><code>struct field asn has json tag but is not exported</code></td><td>

https://github.com/apache/trafficcontrol/blob/61e423f33b8f4b61a6228c9e3d0fbac646d231f2/traffic_ops/traffic_ops_golang/swaggerdocs/v13/asns.go#L51
</td></tr>
<tr><td><code>struct field tag `json:"start_time, omitempty" db:"start_time"` not compatible with reflect.StructTag.Get: suspicious space in struct tag value</code></td><td>

https://github.com/apache/trafficcontrol/blob/61e423f33b8f4b61a6228c9e3d0fbac646d231f2/traffic_ops/traffic_ops_golang/api/async_status.go#L42
</td></tr>
<tr><td><code>struct field IPv4Available repeats json tag "isAvailable" also at event.go:44</code></td><td>

https://github.com/apache/trafficcontrol/blob/61e423f33b8f4b61a6228c9e3d0fbac646d231f2/traffic_monitor/health/event.go#L45
</td></tr>
<tr><td><code>struct field tag `json:"status, omitempty" db:"status"` not compatible with reflect.StructTag.Get: suspicious space in struct tag value</code></td><td>

https://github.com/apache/trafficcontrol/blob/61e423f33b8f4b61a6228c9e3d0fbac646d231f2/traffic_ops/traffic_ops_golang/api/async_status.go#L41
</td></tr>
<tr><td><code>struct field tag `json: "u"` not compatible with reflect.StructTag.Get: bad syntax for struct tag value</code></td><td>

https://github.com/apache/trafficcontrol/blob/61e423f33b8f4b61a6228c9e3d0fbac646d231f2/test/router/server/server.go#L40
</td></tr>
<tr><td><code>struct field tag `json:"id, omitempty" db:"id"` not compatible with reflect.StructTag.Get: suspicious space in struct tag value</code></td><td>

https://github.com/apache/trafficcontrol/blob/61e423f33b8f4b61a6228c9e3d0fbac646d231f2/traffic_ops/traffic_ops_golang/api/async_status.go#L40
</td></tr>
<tr><td><code>struct field tag `json:"end_time, omitempty" db:"end_time"` not compatible with reflect.StructTag.Get: suspicious space in struct tag value</code></td><td>

https://github.com/apache/trafficcontrol/blob/61e423f33b8f4b61a6228c9e3d0fbac646d231f2/traffic_ops/traffic_ops_golang/api/async_status.go#L43
</td></tr>
<tr><td><code>struct field IPv6Available repeats json tag "isAvailable" also at event.go:44</code></td><td>

https://github.com/apache/trafficcontrol/blob/61e423f33b8f4b61a6228c9e3d0fbac646d231f2/traffic_monitor/health/event.go#L46
</td></tr>
<tr><td><code>struct field tag `json:"message, omitempty" db:"message"` not compatible with reflect.StructTag.Get: suspicious space in struct tag value</code></td><td>

https://github.com/apache/trafficcontrol/blob/61e423f33b8f4b61a6228c9e3d0fbac646d231f2/traffic_ops/traffic_ops_golang/api/async_status.go#L44
</td></tr>
</tbody></table>

## Expected behavior:

`go vet -structtag ./...` should find 0 issues.

## Minimal reproduction of the problem with instructions:

```shell
go vet -structtag ./...
```

### PR DESCRIPTION
**Pull request by alebricio**
## What does this PR (Pull Request) do?
- [x] This PR fixes #5763

## Which Traffic Control components are affected by this PR?
- Traffic Ops
- Traffic Monitor
- Traffic Router tester (Go)

## What is the best way to verify this PR?
`go vet -structtag ./...` should find 0 issues.

## The following criteria are ALL met by this PR
- [x] This PR includes tests OR I have explained why tests are unnecessary
- [x] This PR includes documentation OR I have explained why documentation is unnecessary
- [x] This PR includes an update to CHANGELOG.md OR such an update is not necessary
- [x] This PR includes any and all required license headers
- [x] This PR **DOES NOT FIX A SERIOUS SECURITY VULNERABILITY** (see [the Apache Software Foundation's security guidelines](https://www.apache.org/security/) for details)

### CODE REVIEW

File: traffic_monitor/health/event.go [hunk 1]
```diff
@@ -42,8 +42,8 @@ type Event struct {
 	Hostname      string `json:"hostname"`
 	Type          string `json:"type"`
 	Available     bool   `json:"isAvailable"`
-	IPv4Available bool   `json:"isAvailable"`
-	IPv6Available bool   `json:"isAvailable"`
+	IPv4Available bool   `json:"isIPv4Available"`
```
Comment (zrhoffman) [inline traffic_monitor/health/event.go:45]: This should be `ipv4Available` to match the JSON for `tc.IsAvailable.Ipv4Available`:
https://github.com/apache/trafficcontrol/blob/730d842d07d736c344adad9f5081ec30e6790ea9/lib/go-tc/crstates.go#L41

File: traffic_monitor/health/event.go [hunk 2]
```diff
@@ -42,8 +42,8 @@ type Event struct {
...
+	IPv6Available bool   `json:"isIPv6Available"`
```
Comment (zrhoffman) [inline traffic_monitor/health/event.go:46]: This should be `ipv6Available` to match the JSON for `tc.IsAvailable.Ipv6Available`:
https://github.com/apache/trafficcontrol/blob/730d842d07d736c344adad9f5081ec30e6790ea9/lib/go-tc/crstates.go#L42

**Review by zrhoffman — CHANGES_REQUESTED** [review]

File: traffic_monitor/health/event.go (hunk 1 above)
Comment (alebricio) [reply traffic_monitor/health/event.go]: Made changes ~ Thank you for your review!

**Review by zrhoffman — APPROVED** [review]
This all looks good. Thanks!

**Pull request merged**